docs/tutorials/dataassociation/KDTree_Tutorial.py

```diff
@@ -0,0 +1,403 @@
+#!/usr/bin/env python
+# coding: utf-8
+"""
+=================================================
+11 - :math:`k`-d trees and TPR-trees
+=================================================
+"""
+
+# %%
+# :math:`k`-d trees and TPR-trees are both data structures which can be used to index objects in multi-dimensional
+# space.
+# 
+# - A :math:`k`-d tree is a binary search tree that operates in :math:`k` dimensions where each node represents a
+#   :math:`k`-dimensional point, where :math:`k` represents any number of dimensions. A :math:`k`-d tree is useful for
+#   indexing the positions of **static** objects.
```

Comment:
```suggestion
# - A :math:`k`-d tree is a binary search tree that operates in :math:`k` dimensions. Each node represents a
#   :math:`k`-dimensional point, where :math:`k` represents any number of dimensions. A :math:`k`-d tree is useful for
#   indexing the positions of **static** objects.
```
This sentence could be rephrased. The repetition of "where" makes it a bit long-winded.